jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt

```diff
@@ -102,20 +102,41 @@ final class JiraSync(
         issueList: MutableList<String>, imsProject: IMSProject
     ) {
         logger.info("ISSUE LIST $issueList")
+        for (issueId in issueList) {
```

Comment:
could this be split into two methods?
an automatic refactoring should be capable of doing this

Reply:
as we still have to break through multiple levels, this would make it unreadable

Reply:
7a5372a9ec59b96bff03e731bbbf8ecf4a7c7214

Reply:
Thx